src/observer.js

```diff
@@ -177,17 +157,12 @@ function makeComponentReactive(render) {
      * If props are shallowly modified, react will render anyway,
      * so atom.reportChanged() should not result in yet another re-render
      */
-    let skipRender = false
+    this[skipRenderKey] = false
```

Comment:
could we make this a non-enumerable prop?

Reply:
I think yes, of course